src/App.tsx

```diff
@@ -19,6 +19,7 @@ import store, { RootState } from './redux-store';
 import theme from './theme';
 import { AppStackParamList, ROUTE_NAMES } from './TypesAndInterfaces/routes';
 import { AppTabNavigator } from './Widgets/navigation/AppTabNavigator';
+import FirstSignIn from './1-Profile/InitianAccountFlow';
```

Comment:
1. Can this import also be renamed to `InitialAccountFlow`?
2. And rename file to `InitialAccountFlow` too.